perf-common/src/test/java/com/github/projectflink/common/AnalyzeToolTest.java

```diff
@@ -14,13 +14,13 @@
 	public void analyzeLogFile() throws IOException {
 		Result result = AnalyzeTool.analyze("src/test/resources/example-log.gz");
 
-		assertEquals(1111726.308427559, result.throughputs.getMean(), EPSILON);
-		assertEquals(7142857.142857143, result.throughputs.getMax(), EPSILON);
-		assertEquals(23653, result.throughputs.getN());
+		assertEquals(385927.23443877226, result.throughputs.getMean(), EPSILON);
+		assertEquals(4545454.545454545, result.throughputs.getMax(), EPSILON);
+		assertEquals(55437, result.throughputs.getN());
 
-		assertEquals(10585.502395920254, result.latencies.getMean(), EPSILON);
+		assertEquals(3290.572362395614, result.latencies.getMean(), EPSILON);
 
-		assertEquals(4, result.perHostLatancies.size());
-		assertEquals(4, result.perHostThroughputs.size());
+		assertEquals(10, result.perHostLatancies.size());
+		assertEquals(10, result.perHostThroughputs.size());
```

Comment:
I'm assuming you've verified the new values ...

Reply:
Of course :)

But seriously, it doesn't matter. This is more like a regression test and final validation. You have a `input` file and expected `result` of `f(input)`. If you do not change/modify the function `f`, but only update the `input` (like in this case) it's ok to blindly update `result` based on `f`.

But on top of all of that, I did some manual analyzing/testing/validating during benchmarking.